src/curated-corpus/pages/CustomSectionsPage/CustomSectionsPage.tsx

```diff
@@ -0,0 +1,488 @@
+import React, { useEffect, useState } from 'react';
+import { DateTime } from 'luxon';
+import {
+  Section,
+  SectionStatus,
+  useGetScheduledSurfacesForUserQuery,
+  useGetSectionsWithSectionItemsLazyQuery,
+} from '../../../api/generatedTypes';
+import { DropdownOption } from '../../helpers/definitions';
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+  Paper,
+  Chip,
+  TextField,
+  MenuItem,
+} from '@mui/material';
+import { HandleApiResponse } from '../../../_shared/components';
+
+interface GroupedSections {
+  scheduled: Section[];
+  live: Section[];
+  expired: Section[];
+}
+
+export const CustomSectionsPage: React.FC = (): JSX.Element => {
+  const [currentScheduledSurfaceGuid, setCurrentScheduledSurfaceGuid] =
+    useState('');
+  const [scheduledSurfaceOptions, setScheduledSurfaceOptions] = useState<
+    DropdownOption[]
+  >([]);
+  const [groupedSections, setGroupedSections] = useState<GroupedSections>({
+    scheduled: [],
+    live: [],
+    expired: [],
+  });
+
+  // Get sections with section items
+  const [
+    callGetSectionsWithSectionItemsQuery,
+    { data, loading, error, refetch },
+  ] = useGetSectionsWithSectionItemsLazyQuery({
+    fetchPolicy: 'cache-and-network',
+    notifyOnNetworkStatusChange: true,
+    variables: {
+      scheduledSurfaceGuid: currentScheduledSurfaceGuid,
```

Comment:
we can add the `createSource == MANUAL` filter here as well, to retrieve `MANUAL` sections.
lets do the same for the ML sections page.